data/utils.py

```diff
@@ -3,33 +3,52 @@
 import torchvision.transforms as transforms
 import torch
 
+from continuum.datasets import PyTorchDataset
+from continuum import TaskSet
+
 DATA_ROOT_CIFAR = "data/datasets/cifar-10"
 DATA_ROOT_MNIST = "data/datasets/mnist"
 
 
-def load_data(dataset:str='cifar', devset=False):
+def load_data(dataset: str = 'cifar', devset=False):
     """Load CIFAR-10 (training and test set)."""
 
     if dataset == 'cifar':
         transform = transforms.Compose(
             [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
         )
-        trainset = torchvision.datasets.CIFAR10(root=DATA_ROOT_CIFAR, train=True, download=True, transform=transform)
-        testset = torchvision.datasets.CIFAR10(root=DATA_ROOT_CIFAR, train=False, download=True, transform=transform)
+        # if continual:
```

Comment:
Where is `continual` set?

Reply:
@cosmic20 you could just remove the commented code we shouldn't need it